uikit/poidetail/src/main/java/com/sygic/ui/view/poidetail/PoiDetailBottomDialogFragment.kt

```diff
@@ -24,18 +25,34 @@ import com.sygic.ui.view.poidetail.viewmodel.SHOWCASE_BEHAVIOR_STATE
 
 private const val POI_DATA = "poi_data"
 
-class PoiDetailBottomDialogFragment : AppCompatDialogFragment() {
+/**
+ * A [PoiDetailBottomDialogFragment] is a custom version of the [DialogFragment] that shows a bottom sheet using custom
+ * [BottomSheetDialog] instead of a floating dialog. It can be used for a visual representation of the [PoiData] object.
+ *
+ * You can register an [DialogFragmentListener] using [setListener] method. Then you will be notified when dialog is dismissed.
+ *
+ * Content colors can be changed with the custom style or attribute. See "Sample app" for more info.
```

Comment:
nedaval by som See Sample app, ked to moze clovek pouzivat uplne nezavisle. Nebude vediet mozno ani kde by ju mal hladat...

```suggestion
 * Content colors can be changed with the custom style or attributes.
``

Pripadne uviest tam nejake attributy, ktore sa daju pouzit na customizaciu a meno toho stylu co sa da definovat. Ten tu nevidim 🤔